### DISCUSSION TITLE
[1.16.5] Cutting recipe names inconsistent with vanilla crafting recipe conventions

### DISCUSSION THREAD

**Discussion by ShockMicro** [vectorwing/FarmersDelight#744] [Ideas]
This is completely a minor nitpick, but should definitely help with understanding how the cutting recipes work.
- Stripping recipes such as acacia wood are called `acacia_wood` instead of `stripped_acacia_wood` which could lead to the wrong conclusion that the recipe is crafting the normal version of wood.
- This similarly happens with slices, like with the salmon being called `cooked_salmon` instead of `cooked_salmon_slice`.
- Vanilla handles multiple recipes having the same ingredient by giving them distinct names. Cutting does not follow this convention, which wrongly led me to believe you could cut something into a crimson door. This can be fixed by using vanilla's conventions for this as well, with names such as `crimson_planks_from_door` or `crimson_planks_from_sign`. 
Feel free to close if you wish, as this does not affect gameplay.

**Comment by ShockMicro** [comment]
I see now that you use datagen for this, so I guess it's probably a bit more complex than I thought to make those changes. Whoops.

**Comment by vectorwing** [comment]
I originally decided to follow this input-based convention due to the nature of the Cutting Board as an "uncrafting table", as it frequently gives the same outputs for many recipes. Quite a lot of them would be "output_from_x" from this new nomenclature, and since inputs were the more unique part of the recipe, I assumed it would be easier to search for them.

Now, I do agree with the readability issues presented; just not sure whether to adopt the vanilla standard. Perhaps prefixes could work instead, like "recycling_x" or similar, with unique cutting pairs (e.g. Cabbage -> Cabbage Leaf) being output-named. Might think more about it later down.

As for datagen, it would be mostly trivial on my end (one of datagen's many advantages is automation), though I can see how a PR about it might be a lot of work. 😅
